LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2018 The eth-typing contributors
+Copyright (c) 2019 The Ethereum Foundation
```

Comment:
Nitpick: I find  *`<year>` The `<project-name>` contributors*
Slightly more inclusive and hence welcoming to non EF contributors. But maybe this has legal implications that I'm not aware of.

Reply:
I don't have a strong feeling about one or the other. The inclusiveness argument has merit to me. But my strongest feeling is that we should have consistency.

I would probably approve a PR to the templates repo that changes this to "The <PROJECT_NAME> contributors". :)

Reply:
>I would probably approve a PR to the templates repo that changes this to "The <PROJECT_NAME> contributors". :)

I take your word for it.

https://github.com/ethereum/ethereum-python-project-template/pull/29